src/components/AutoComplete/AutoComplete.tsx

```diff
@@ -0,0 +1,721 @@
+        return childrenToComboboxItemArray(child.props.children, callback, heading);
+      }
+    }
+    return [];
+  });
+};
+const SelectNoDataContainer = styled.div`
+  border: none;
+  display: block;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: left;
+  cursor: default;
+  &[hidden="true"] {
+    display: none;
+  }
+  ${({ theme }) => `
+    font: ${theme.click.genericMenu.button.typography.label.default}
+    padding: ${theme.click.genericMenu.button.space.y} ${theme.click.genericMenu.item.space.x};
+    background: ${theme.click.genericMenu.button.color.background.default};
+    color: ${theme.click.genericMenu.button.color.label.default};
+  `}
+`;
+const StyledSelectTrigger = styled(Trigger)<{ $error: boolean }>`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+
+  span:first-of-type {
+    max-width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  ${({ theme, $error }) => `
+    border-radius: ${theme.click.field.radii.all};
+    padding: ${theme.click.field.space.y} ${theme.click.field.space.x};
+    gap: ${theme.click.field.space.gap};
+    font: ${theme.click.field.typography.fieldText.default};
+    color: ${theme.click.field.color.text.default};
+    border: 1px solid ${theme.click.field.color.stroke.default};
+    background: ${theme.click.field.color.background.default};
+    &:hover {
+      border: 1px solid ${theme.click.field.color.stroke.hover};
+      background: ${theme.click.field.color.background.hover};
+      color: ${theme.click.field.color.text.hover};
+    }
+    ${
+      $error
+        ? `
+      font: ${theme.click.field.typography.fieldText.error};
+      border: 1px solid ${theme.click.field.color.stroke.error};
+      background: ${theme.click.field.color.background.active};
+      color: ${theme.click.field.color.text.error};
+      &:hover {
+      border: 1px solid ${theme.click.field.color.stroke.error};
+      color: ${theme.click.field.color.text.error};
+      }
+    `
+        : `
+    &:focus,
+    &[data-state="open"] {
+      font: ${theme.click.field.typography.fieldText.active};
+      border: 1px solid ${theme.click.field.color.stroke.active};
+      background: ${theme.click.field.color.background.active};
+      color: ${theme.click.field.color.text.active};
+      & ~ label {
+        color: ${theme.click.field.color.label.active};
+        font: ${theme.click.field.typography.label.active};;
+      }
+    }
+    `
+    };
+    &:disabled {
+      font: ${theme.click.field.typography.fieldText.disabled};
+      border: 1px solid ${theme.click.field.color.stroke.disabled};
+      background: ${theme.click.field.color.background.disabled};
+      color: ${theme.click.field.color.text.disabled};
+      cursor: not-allowed;
+    }
+  `}
+  [data-hide-in-trigger] {
+    display: none;
+  }
+`;
```

Comment:
this can be deleted. The style should be that of the searchfield component, see the comment below